crates/bevy_pbr/src/render/morph.rs

```diff
@@ -89,7 +90,9 @@ pub fn extract_morphs(
         add_to_alignment::<f32>(&mut uniform.buffer);
 
         let index = (start * mem::size_of::<f32>()) as u32;
-        values.push((entity, MorphIndex { index }));
+        // NOTE: Because morph targets require per-morph target texture bindings, they cannot
+        // currently be batched.
+        values.push((entity, (MorphIndex { index }, NoAutomaticBatching)));
```

Comment:
Seems like it would be safer to make "no automatic batching" the default and then _opt in_ to automatic batching via a marker type. Is it done in reverse for perf reasons?

Reply:
You could say that. I want people to get better performance by default. Why not batch if you can? And then if stuff breaks, opt-out or fix the things needed to get batching working.

Reply:
I think "automatic batching" should definitely apply to user facing types for things like PbrBundle, SpriteBundle, etc. But if someone is building a new rendered entity type with custom rendering logic, idk if we should batch that by default if there are "hidden" data access constraints that we don't check or enforce. Provided it doesn't cost us anything significant for things in the hot path, I think it makes sense to opt-in things like sprites, 2d + 3d meshes, etc.